config/config.example.yml

```diff
@@ -350,6 +350,8 @@ item:
     # Rounded to the nearest size in the list of selectable sizes on the
     # settings menu.  See pageSizeOptions in 'pagination-component-options.model.ts'.
     pageSize: 5
+    # Show the bitstream access status label
```

Comment:
Just for clarity, we might want to have this say "Show the bitstream access status label on the item page".  (That just clarifies it's not shown anywhere else at this time)